docs/topics/exporters.rst

```diff
@@ -311,6 +311,10 @@ CsvItemExporter
       multi-valued fields, if found.
    :type include_headers_line: str
 
+    :param errors: The optional string that specifies how encoding and decoding errors are to be handled.
+        For more information see `documentation <https://docs.python.org/3/library/io.html#io.TextIOWrapper>`_.
```

Comment:
Please try building the documentation locally with `tox -e docs` to verify, but I think using InterSphinx here should work:

```suggestion
        For more information see :class:`io.TextIOWrapper`.
```